.changeset/blue-days-nail.md

```diff
@@ -0,0 +1,7 @@
+---
+"@justeattakeaway/pie-button": minor
```

Comment:
Would you mind removing this entry for pie-button? It doesn't warrant a new npm publish IMO, since it's an internal repo change.